scripts/babel-6/inline-requires-test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+jest.autoMockOff();
+
+var babel = require('babel-core');
+
+describe('inline-requires', function() {
+  it('should inline single usage', function() {
+    compare([
+      'var foo = require("foo");',
+      'foo.bar()'
+    ], [
+      'require("foo").bar();'
+    ]);
+  });
+
+  it('should inline requires that are not assigned', function() {
```

Comment:
Yea, this was a terrible test name, I just kept going with the `should inline...`, sorry :(